src/EditorFeatures/Test/SolutionCrawler/WorkCoordinatorTests.cs

```diff
@@ -1624,7 +1624,7 @@ private void Process(DocumentId _, CancellationToken cancellationToken)
                     this.RunningEvent.Set();
 
                     // Wait until unblocked
-                    this.BlockEvent.Wait();
+                    this.BlockEvent.Wait(cancellationToken);
```

Comment:
📝 It's highly likely that changes to test synchronization are going to break the test semantics at least once